kahuna/public/js/search/query.js

```diff
@@ -83,9 +94,13 @@ query.controller('SearchQueryCtrl',
 
     $scope.$watchCollection(() => ctrl.filter, onValChange(filter => {
         filter.uploadedBy = filter.uploadedByMe ? ctrl.user.email : undefined;
-        $state.go('search.results', filter);
+        //$state.go('gr.search.results', filter);
```

Comment:
This, at the moment doesn't filter down to `results` as results requires a reload to take affect, which is massively inefficient.